OmsiHook/OmsiComplMapObjInst.cs

```diff
@@ -128,5 +130,70 @@ public OmsiCollFeedback[] CollFeedbacks
         {
             get => Memory.ReadMemoryStructArray<OmsiCollFeedback>(Address + 0x254);
         }
+        
+        /// <summary>
+        /// Get a float variable for an object from its name.
+        /// </summary>
+        /// <param name="VarName">Variable Name</param>
+        /// <returns>requested float value</returns>
+        /// <exception cref="System.Exception"/>
+        public float GetVariable(string VarName)
+        {
+            this.PublicVars.UpdateFromHook();
+            this.PublicVars.UpdateFromHook();
+            int index = this.ComplMapObj.GetVarIndex(VarName);
+            if (index < this.PublicVars.arrayCache.Length && index >= 0)
+                return this.PublicVars[index].Float;
+            else
+                throw new Exception("Variable '" + VarName + "' not found in object. - Index Out Of Bounds");
+        }
+
+        /// <summary>
+        /// Set a float variable for an object to a value from its name.
+        /// </summary>
+        /// <param name="VarName">Variable Name</param>
+        /// <param name="Value">Desired Value</param>
+        /// <exception cref="System.Exception"/>
+        public void SetVariable(string VarName, float Value)
+        {
+            this.PublicVars.UpdateFromHook();
+            int index = this.ComplMapObj.GetVarIndex(VarName);
+            if (index < this.PublicVars.arrayCache.Length && index >= 0)
+                this.PublicVars[index] = new() { Float = Value };
```

Comment:
Maybe write a proper constructor for this (both a 0 param and a 1 param to make this sort of stuff neater)

Reply:
Ergo for string ptrs.